.github/workflows/winget.yml

```
@@ -0,0 +1,20 @@
+name: Publish to WinGet
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  publish:
+    runs-on: windows-latest # Action can only be run on windows
+    steps:
+      - name: Extract version from tag
+        id: extract_version
+        run: |
+          $version = [regex]::Match("${{ github.event.release.tag_name }}", "(?<=v)[\d.]+").Value
+          Write-Output "version=$version" >> $env:GITHUB_OUTPUT
+      - uses: vedantmgoyal2009/winget-releaser@v2
```

```suggestion
      - uses: vedantmgoyal2009/winget-releaser@3b55acc8e47059aba464e3ce9a230e60335a651a # v2
```
Suggest referencing the specific commit instead of the v2 tag as a security best practice

---

The problem with this is that you would have to update the action often when it breaks (possibly due to upstream changes in winget-pkgs). Are you okay with that?

---

Yes - we'll update it manually when needed